test/ui-automation/test/helpers/wallet.js

```diff
@@ -320,3 +344,47 @@ async function _updatePreferences() {
 
     // TODO validate success message
 }
+
+async function _addNewVault(vaultName) {
+    // User click on Add Vault button
+    await browser.refresh();
+    await browser.pause(9000);
+    const addVaultButton = await $("#add-new-vault-button");
+    await addVaultButton.click();
+    await _vaultNameInput(vaultName);
+    await _createVault();
+}
+
+async function _vaultNameInput(vaultName){
+    // User enters vault name
+    const addVaultInput = await $("#input-VaultName");
+    await addVaultInput.click();
+    await addVaultInput.setValue(vaultName);
+}
+
+async function _createVault(){
+    const addAction = await $('.btn-primary*=Add')
+    await addAction.click();
+}
+
+async function _validationError(msg){
+    const errorMsg = await $('.text-primary-vampire')
```

Comment:
This is not ideal to select element by the the colour class since it can change. Could you add an id to it and use that instead?